samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -47,19 +47,22 @@ static const char location_usage_str[] =
 	"  cancel:   Cancel/stop on going request. No options\n";
 
 static const char location_get_usage_str[] =
-	"Usage: location get [--mode <mode>] [--method <method>] [--interval <secs>]\n"
+	"Usage: location get [--mode <mode>] [--method <method>]\n"
+	"[--timeout <secs>] [--interval <secs>]\n"
```

Comment:
`--timeout` is still given as milliseconds, right?

Reply:
yep, same bug in all timeouts/interval. I'll fix them